### DISCUSSION TITLE
Google pay option is not showing in default sheet

### DISCUSSION THREAD

**Discussion by pinal-dhaduk** [flutter-stripe/flutter_stripe#967] [Q&A]
**Describe the bug**
I have used flutter_stripe SDK 5.0.0 and and I am trying to implement google pay functionality in default sheet. also I have enable google pay option inside SetupPaymentSheetParameters

          await Stripe.instance
              .initPaymentSheet(
            paymentSheetParameters: SetupPaymentSheetParameters(
              // Main params
              paymentIntentClientSecret:
                  apiPaymentResponse.data!.data!.paymentIntent,
              merchantDisplayName: 'Weguide Stripe Store ',
              // Customer params
              customerId: apiPaymentResponse.data!.data!.customer,
              customerEphemeralKeySecret:
                  apiPaymentResponse.data!.data!.ephemeralKey,
              // Extra params
              applePay: const PaymentSheetApplePay(
                merchantCountryCode: 'US',
              ),
              googlePay: const PaymentSheetGooglePay(
                  merchantCountryCode: 'US',
                  testEnv: true,
                  currencyCode: "USD"),
              style: ThemeMode.system,
              billingDetails: billingDetails,
            ),
          )
              .then((value) async {
            confirmPayment(
              context: context,
              state: state,
            );
            if (state.mounted) {
              notifyListeners();
            }
          });

**To Reproduce**
1. create a keys from backend
2. initialise payment sheet initPaymentSheet
3. Open payment sheet

**Expected behavior**
When I open stripe default sheet at that time I need google pay option in default sheet.

**Smartphone / tablet**
 - Device: [Galaxy s10]
 - OS: [Android 12]
 - Package version: [5.0.0]
 - Flutter version [3.3.0]
 
 **Additional context**
![Screenshot 2022-09-12 at 6 08 29 PM](https://user-images.githubusercontent.com/29156738/189655871-122e2f7b-b459-4bac-9a4e-a92fc3b0782b.png)

**Comment by jonasbark** [comment]
What happens when you call
```
final googlePaySupported = await Stripe.instance.isGooglePaySupported(IsGooglePaySupportedParams());
```
like in https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/wallets/google_pay_stripe_screen.dart ?

**Comment by lohitbura07** [comment]
@jonasbark  I am facing the same problem 

here is my code

await Stripe.instance.initPaymentSheet(
paymentSheetParameters: SetupPaymentSheetParameters(
allowsDelayedPaymentMethods: true,
merchantDisplayName: 'MRGO',
setupIntentClientSecret: setupIntentModel.setupIntent?.first,
googlePay: const PaymentSheetGooglePay(
merchantCountryCode: 'DE',
testEnv: true,
currencyCode: 'EUR'),
));

await Stripe.instance.presentPaymentSheet();

final googlePaySupported = await Stripe.instance.isGooglePaySupported(IsGooglePaySupportedParams());

This is returning me True

**Comment by pinal-dhaduk** [comment]
> What happens when you call
> 
> ```
> final googlePaySupported = await Stripe.instance.isGooglePaySupported(IsGooglePaySupportedParams());
> ```
> 
> like in https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/wallets/google_pay_stripe_screen.dart ?

Its returning true. value
